blocks/content-band-checkmarks/content-band-checkmarks.css

```diff
@@ -0,0 +1,130 @@
+.content-band-checkmarks > div > div:first-child > p:first-child {
+    font-family: var(--sans-serif-font-medium);
+    font-size: var(--font-size-11);
+    font-weight: var(--font-weight-medium);
+    line-height: var(--line-height-120);
+    letter-spacing: var(--letter-spacing-02-em);
+    margin: 0 0 var(--spacer-element-07) 0;
+}
+
+.content-band-checkmarks > div > div:first-child > h2 {
+    font-family: var(--serif-font);
+    font-size: var(--font-size-32);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-120);
+    letter-spacing: var(--letter-spacing-1-75);
+    margin: 0 0 var(--spacer-element-08) 0;
+}
+
+.content-band-checkmarks > div > div:first-child > p:not(:first-child):not(.button-container) {
+    font-family: var(--sans-serif-font-light);
+    font-size: var(--font-size-16);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-160);
+    letter-spacing: var(--letter-spacing-001-em);
+    margin: 0 0 var(--spacer-element-08) 0;
+}
+
+.content-band-checkmarks > div > div:first-child > .button-container {
+    margin: 0 0 var(--spacer-layout-04);
+}
+
+.content-band-checkmarks > div > div:first-child > .button-container>strong>a {
+    font-family: var(--sans-serif-font-medium);
+    font-size: var(--font-size-16);
+    font-weight: var(--font-weight-medium);
+    line-height: var(--line-height-160);
+    letter-spacing: 0.16px;
+    padding: 16px var(--spacer-element-07);
+}
+
+.content-band-checkmarks > div > div:nth-child(2) > ul {
+    margin: 0;
+    padding: 0;
+}
+
+.content-band-checkmarks > div > div:nth-child(2) > ul>li {
+    font-family: var(--sans-serif-font-medium);
+    font-size: var(--font-size-18);
+    font-weight: var(--font-weight-medium);
+    line-height: var(--line-height-160);
+    letter-spacing: var(--letter-spacing-001-em);
+    margin: 0 0 16px;
+    padding: 0 24px;
+    height: 48px;
+    display: flex;
+    align-items: center;
+}
+
+main > .section.section-header.list-section.bg-neutral-bone > .content-band-checkmarks-wrapper > .content-band-checkmarks > div > div:nth-child(2) > ul>li {
+    background-color: white;
+}
+
+.content-band-checkmarks > div > div:nth-child(2) > ul>li::before {
+    margin-right: 24px;
+}
+
+
+/* Tablet CSS */
+@media (min-width: 768px) {
+    .content-band-checkmarks > div > div:first-child > h2 {
+        max-width: 516px;
+    }
+
+    .content-band-checkmarks > div > div:first-child > p:not(:first-child):not(.button-container) {
+        max-width: 517px;
```

Comment:
Let's keep the max-width value to `516px` so that we don't have just one pixel off.

```
.content-band-checkmarks > div > div:first-child > h2,
.content-band-checkmarks > div > div:first-child > p:not(:first-child):not(.button-container) {
   max-width: 516px;
}
```